root/usr/bin/nm-configure

```diff
@@ -49,13 +51,13 @@ autoconnect-priority=1
 [802-3-ethernet]
 mac-address=$mac
 [ipv4]
-method=manual
+method=$ipv4_method
 address1=$ip
 gateway=$gw
```

Comment:
Since we're changing the default from manual to auto, how does networkmanager behave when we pass address and gateway, but at the same time set method to auto?

Reply:
Good point, this is an overlook. This needs to be manual.

It brings another missed feature - IPv6 is not yet done for manual mode (when you attach the ISO and use TUI to configure IP). So it is impossible to reach to this state yet.

Reply:
Corrected, however as I said this command is called from the text user interface and the screen does not yet have elements to work with.